carbon/app.py

```diff
@@ -16,83 +16,148 @@
-        .select_from(persons.outerjoin(orcids).join(dlcs)) \
-        .where(persons.c.EMAIL_ADDRESS != None) \
-        .where(persons.c.LAST_NAME != None) \
-        .where(persons.c.KRB_NAME_UPPERCASE != None) \
-        .where(persons.c.KRB_NAME_UPPERCASE != 'UNKNOWN') \
-        .where(persons.c.MIT_ID != None) \
-        .where(persons.c.ORIGINAL_HIRE_DATE != None) \
-        .where(persons.c.APPOINTMENT_END_DATE >= datetime(2009, 1, 1)) \
-        .where(func.upper(dlcs.c.ORG_HIER_SCHOOL_AREA_NAME).in_(AREAS)) \
-        .where(persons.c.PERSONNEL_SUBAREA_CODE.in_(PS_CODES)) \
-        .where(func.upper(persons.c.JOB_TITLE).in_(TITLES))  # noqa: E711
+    sql = (
+        select(
+            persons.c.MIT_ID,
+            persons.c.KRB_NAME_UPPERCASE,
+            persons.c.FIRST_NAME,
+            persons.c.MIDDLE_NAME,
+            persons.c.LAST_NAME,
+            persons.c.EMAIL_ADDRESS,
+            persons.c.DATE_TO_FACULTY,
+            persons.c.ORIGINAL_HIRE_DATE,
+            dlcs.c.DLC_NAME,
+            persons.c.PERSONNEL_SUBAREA_CODE,
+            persons.c.APPOINTMENT_END_DATE,
+            orcids.c.ORCID,
+            dlcs.c.ORG_HIER_SCHOOL_AREA_NAME,
+            dlcs.c.HR_ORG_LEVEL5_NAME,
+        )
+        .select_from(persons)
+        .outerjoin(orcids)
+        .join(dlcs)
+        .where(persons.c.EMAIL_ADDRESS.is_not(None))
+        .where(persons.c.LAST_NAME.is_not(None))
+        .where(persons.c.KRB_NAME_UPPERCASE.is_not(None))
+        .where(persons.c.KRB_NAME_UPPERCASE != "UNKNOWN")
+        .where(persons.c.MIT_ID.is_not(None))
+        .where(persons.c.ORIGINAL_HIRE_DATE.is_not(None))
+        .where(persons.c.APPOINTMENT_END_DATE >= datetime(2009, 1, 1))
+        .where(func.upper(dlcs.c.ORG_HIER_SCHOOL_AREA_NAME).in_(AREAS))
+        .where(persons.c.PERSONNEL_SUBAREA_CODE.in_(PS_CODES))
+        .where(func.upper(persons.c.JOB_TITLE).in_(TITLES))
+    )
     with closing(engine().connect()) as conn:
-        for row in conn.execute(sql):
-            yield dict(zip(row.keys(), row))
+        result = conn.execute(sql)
+        for row in result:
+            yield dict(zip(result.keys(), row))
 
 
 def articles():
     """An article generator.
 
     Returns an iterator over the AA_ARTICLE table.
     """
-    sql = select([aa_articles]) \
-        .where(aa_articles.c.ARTICLE_ID != None) \
-        .where(aa_articles.c.ARTICLE_TITLE != None) \
-        .where(aa_articles.c.DOI != None) \
-        .where(aa_articles.c.MIT_ID != None)  # noqa: E711
+    sql = (
+        select(aa_articles)
+        .where(aa_articles.c.ARTICLE_ID.is_not(None))
+        .where(aa_articles.c.ARTICLE_TITLE.is_not(None))
+        .where(aa_articles.c.DOI.is_not(None))
+        .where(aa_articles.c.MIT_ID.is_not(None))
+    )
     with closing(engine().connect()) as conn:
-        for row in conn.execute(sql):
-            yield dict(zip(row.keys(), row))
+        result = conn.execute(sql)
```

Comment:
Same here as above

Reply:
Same response as above. :)

Reply:
@ehanson8 and I had a conversation offline and agreed to keep it as-is for now!